monitoring/api/v3/uptime-check-client/snippets.py

```diff
@@ -23,10 +23,10 @@
 
 
 # [START monitoring_uptime_check_create]
-def create_uptime_check_config(project_name, host_name=None,
```

Comment:
@GoogleCloudPlatform/python-samples-owners Is it preferred to have one function per region tag, or is doubling up OK?

Reply:
Personally I'm ok with either way.